pkg/kmsg/events.go

```diff
@@ -0,0 +1,45 @@
+package kmsg
+
+import (
+	"os"
+	"runtime"
+
+	"github.com/leptonai/gpud/pkg/log"
+)
+
+// CreateEventsWatcher creates a new events watcher that will watch the kmsg.
+// Experimental.
+// TODO: integrate with the events store.
+func CreateEventsWatcher(matchFunc func(line string) (eventName string, message string)) (Watcher, error) {
+	if os.Getenv("GPUD_EXPERIMENTAL_KMSG_WATCHER") != "true" {
```

Comment:
probably we can remove the feature-gate,  and don't fatal the process if  failed to start watch.